stylesheets/pages/cart.css

```diff
@@ -0,0 +1,101 @@
+.cart-wrapper {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;

```

Comment:
You might like to use utility classes for these coming straight from your component library.

Reply:
Yeah, I will refactor this in the dev branch.